packages/example/pages/index.tsx

```diff
@@ -250,6 +251,70 @@ const Example = ({ authEnabled }: AppContextProps) => {
         </ConnectButton.Custom>
       </div>
 
+      <div>
+        <h3 style={{ fontFamily: 'sans-serif' }}>Rainbow button</h3>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            gap: 10,
+          }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+            }}
+          >
+            <RainbowButton />
+          </div>
+        </div>
+      </div>
+
+      <div>
+        <h3 style={{ fontFamily: 'sans-serif' }}>Rainbow button (custom)</h3>
+        <div
```

Comment:
We could bump this up to their "Custom buttons" section. We can probably skip the styling too just to reduce testable surface area